### PR TITLE
interface: various fixes for new groups layout 

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
A host of changes to address design criticism and QA discoveries. Highlights:
- Virtualised, searchable contact list
- Return of new/join groups screen (now with 200% more indigo)
- Made invitePopover match figma design, including an email field for inviting with urbit ID
- iOS scrolling is now working
- No longer weird empty space at bottom of screen on iOS safari

### CODE REVIEW

File: pkg/interface/src/views/apps/launch/components/tiles/tile.js [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
     if (to) {
-      if (routeList.indexOf(to) === -1) {
-        childElement= (<Link to={to}>{childElement}</Link>);
-      } else {
-        childElement= (<a href={to}>{childElement}</a>);
-      }
-      
+      childElement= (<Link to={to}>{childElement}</Link>);
+    } else if (href) {
+      childElement= (<a href={href}>{childElement}</a>);
```
Comment (matildepark) [inline pkg/interface/src/views/apps/launch/components/tiles/tile.js]: Should preserve the old logic here. It’s for third party apps which need to be a href’d.

Comment (matildepark) [reply pkg/interface/src/views/apps/launch/components/tiles/tile.js]: Or are you just passing href for those separately

File: pkg/interface/src/views/apps/publish/components/NotebookRoutes.tsx [hunk 2]
```diff
@@ -70,6 +71,9 @@ export function NotebookRoutes(
           const { noteId } = routeProps.match.params;
           const note = notebook?.notes[noteId];
           const noteUrl = relativePath(`/note/${noteId}`);
+          if(!note) {
+            return <Center height="100%"><LoadingSpinner /></Center>;
+          }
```
Comment (matildepark) [inline pkg/interface/src/views/apps/publish/components/NotebookRoutes.tsx:76]: The only thing in this commit is spinner behaviour for no note case? But the note never loads in, last I looked

File: pkg/interface/package.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
     "react-oembed-container": "^1.0.0",
     "react-router-dom": "^5.0.0",
     "react-virtuoso": "^0.20.0",
+    "react-visibility-sensor": "^5.1.1",
```
Comment (tylershuster) [inline pkg/interface/package.json:38]: Is this needed?

**Review by matildepark — APPROVED** [review]
Tweaked a few regressions. In we go!

**Pull request merged**